en/sim_gazebo_gz/README.md

```diff
@@ -218,16 +283,35 @@ To add a new model:
      PX4_SYS_AUTOSTART=<your new airframe id> ./build/px4_sitl_default/bin/px4
      ```
 
+1. Add CMake Target for the [airframe](https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/init.d-posix/airframes/CMakeLists.txt).
+
+   - If you plan to use "regular" mode, add your model SDF to `Tools/simulation/gz/models/`.
+   - If you plan to use _standalone_ mode, add your model SDF to `~/.simulation-gazebo/models/`
+
+   You can of course also use both.
+
+### Adding a World
+
+To add a new world:
+
+1. Add your world to the list of worlds found in the [`CMakeLists.txt` here](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/simulation/gz_bridge/CMakeLists.txt).
```

Comment:
Same comment as one above - this slightly feels like it is mixing "adding a world for everyone" and "adding a world for local testing".